### PR TITLE
Add Remark command and functionality

### PR DESCRIPTION
**Pull request by EssWhyy**

### CODE REVIEW

File: src/test/java/socialite/model/person/RemarkTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package socialite.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class RemarkTest {
+
```
Comment (david-eom) [inline src/test/java/socialite/model/person/RemarkTest.java:9]: Actually in hindsight I realised that maybe we should include a test to make sure remark is not an empty string containing only whitespaces i.e. `"   "`. We can include this in future commits.
